pkg/admission/admit_job.go

```diff
@@ -173,6 +173,13 @@ func validateTaskTemplate(task v1alpha1.TaskSpec, job v1alpha1.Job, index int) s
 	var coreTemplateSpec k8score.PodTemplateSpec
 	k8scorev1.Convert_v1_PodTemplateSpec_To_core_PodTemplateSpec(&v1PodTemplate.Template, &coreTemplateSpec, nil)
 
+	// Skip verify container SecurityContex.Privileged
+	for i, container := range coreTemplateSpec.Spec.Containers {
+		if container.SecurityContext != nil && container.SecurityContext.Privileged != nil {
+			coreTemplateSpec.Spec.Containers[i].SecurityContext.Privileged = nil
```

Comment:
Where did we validate `privileged`, `ValidatePodTemplate`? BTW, we should enable/disable this validation according to `--allow-privileged` of kube-apiserver.

Reply:
Yes. Just leave the validation to apiserver, as we may not know the cluster flag.

Reply:
> Yes. Just leave the validation to apiserver, as we may not know the cluster flag.

Better to add this into code notes.

Reply:
is there any other validation dependent on api-server parameter?

Reply:
Let me  have an investigate

Reply:
Should have no other fields. But will ignore some fields validation which depends on the feature gate, like `CustomPodDNS`.   

I think this still does not matter as #412 in.

Reply:
hm... so how to identify those issues before user?

Reply:
@hzxuzhonghu , let's get this merged firstly, and open an issue on how to identify those issues before user;

Reply:
I suggest introducing k8s native ut cases into volcano, i think they should already cover corner cases.